src/chesster.js

```diff
@@ -1064,7 +1064,7 @@ function(bot, message) {
         if(
             !_.isEqual(result.white.id, message.user) &&
             !_.isEqual(result.black.id, message.user) &&
-            !message.player.isModerator()
+            !message.player.isModerator(message)
```

Comment:
it compares it to the league associated with the message.

a little odd - just a helper on the player. Im not sure why or when we added it. but there it is.